theme/tests/multiplatform.py

```diff
@@ -168,17 +169,94 @@ def _create_resource_helper(self, upload_file_path, resource_title=None):
             return resource_title
 
         def test_register_account(self):
+            usr = {
+                'fn': '<strong>First</strong>',
+                'ln': 'last',
+                'email': 'u@example.com',
+                'user': 'uname',
+                'pass': 'drowssap',
+                'org': 'My University',
+                'title': 'Adjunct'
+             }
+
             login_link_css_selector = 'a[href="{}"]'.format(reverse('login'))
             for e in self.driver.find_elements(By.CSS_SELECTOR, login_link_css_selector):
                 if e.is_displayed():
                     e.click()
                     break
 
-            self.assertTrue(reverse('login') in self.driver.current_url)
             # there is nothing to reverse() for sign-up because it is hard coded via Mezzanine
             self.wait_for_visible(By.CSS_SELECTOR, 'a[href="/sign-up/?next="]').click()
             self.wait_for_visible(By.CSS_SELECTOR, '#form-signup')
 
+            # monkey patch SignUpForm to always verify the captcha
+            from theme.forms import SignupForm
+            def verify_captcha_patch(*args, **kwargs):
+                return True
+            SignupForm.verify_captcha = verify_captcha_patch
+
+            # Register a new account
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="first_name"]').send_keys(usr['fn'])
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="last_name"]').send_keys(usr['ln'])
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="email"]').send_keys(usr['email'])
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="username"]').send_keys(usr['user'])
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="password1"]').send_keys(usr['pass'])
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="password2"]').send_keys(usr['pass'])
+            self.wait_for_visible(By.CSS_SELECTOR, 'input[name="recaptcha_response_field"]').send_keys('response')
+            self.wait_for_visible(By.CSS_SELECTOR, 'input#signup').click()
+
+            # Verify new account by clicking on link in that came in email
+            self.wait_for_visible(By.CSS_SELECTOR, '#home-page-carousel')
+            alert_div =  self.wait_for_visible(By.CSS_SELECTOR, '.page-tip')
+            alert_text = alert_div.find_element(By.CSS_SELECTOR, 'p').text
+            self.assertIn('A verification email has been sent', alert_text)
+
+            self.assertEqual(len(mail.outbox), 1)
+            url = re.search(r'(?P<url>https?://[^\s]+)', mail.outbox[0].body).groupdict()['url']
+            user = User.objects.get(email=usr['email'])
+            self.assertFalse(user.is_active)
+            with self.assertRaises(ValueError):
+                _ = user.userprofile.cv.url
```

Comment:
I was just trying to use the with-block syntax for assertRaises and figured that since the throwaway variable is only local to the block it's not too harmful to have an extra variable hanging around. I didn't want to cause any pep or flake errors due to a statement that didn't do anything by merely referencing the property alone on a line which could have instead served the purpose of raising the error.

I'll try to do it more cleanly without the with block and throwaway variable.